static/skin/taskbar.js

```diff
@@ -3,6 +3,39 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
+function setupAutoHidingOfTheToolbar() {
+  let lastScrollTop = 0;
+  let delta = 5;
+  let didScroll = false;
+  const kiwixToolBar = document.querySelector('#kiwixtoolbar');
+
+  window.onscroll = function (e) {
+    didScroll = true;
+  }
```

Comment:
This cancels the previous `onscroll` handler (if any)

Reply:
used an eventHandler instead